### PR TITLE
lookup: remove flaky entries

### PR DESCRIPTION
**Pull request by BridgeAR**
This is actually meant as a test. I would like to see a couple of citgm runs with all these entries removes to actually know if we are able to remove a couple of those entries. It is not meant to actually land.

### CODE REVIEW

File: lib/lookup.json [hunk 1]
```diff
@@ -74,29 +71,25 @@
...
     "maintainers": "jprichardson"
   },
   "body-parser": {
-    "flaky": "aix",
     "maintainers": "dougwilson"
   },
   "uglify-js": {
     "prefix": "v",
-    "flaky": ["ppc", "win32", "v7", "darwin"],
+    "flaky": ["v7"],
```
Comment (MylesBorins) [inline lib/lookup.json]: we can likely remove versions that are no longer maintained correct?

File: lib/lookup.json [hunk 2]
```diff
@@ -420,17 +388,14 @@
...
     "maintainers": "tj"
   },
   "spawn-wrap": {
     "prefix": "v",
-    "flaky": ["win32", "sles", "aix", "darwin"],
     "maintainers": "isaacs"
   },
   "node-report": {
     "prefix": "v",
-    "flaky": "win32",
```
Comment (richardlau) [inline lib/lookup.json]: https://github.com/nodejs/citgm/pull/541

**Comment by targos** [comment]
Needs a rebase

**Comment by BridgeAR** [comment]
I am closing this for now. I'll have another look at the flakes soon again.

**Pull request closed without merging**